README.md

```diff
@@ -583,8 +585,6 @@ In addition, uv respects the following environment variables:
 - `FISH_VERSION`: Used to detect the use of the Fish shell.
 - `BASH_VERSION`: Used to detect the use of the Bash shell.
 - `ZSH_VERSION`: Used to detect the use of the Zsh shell.
-- `RAYON_NUM_THREADS`: Used to control the number of threads used when unzipping and installing
-  packages. See the [rayon documentation](https://docs.rs/rayon/latest/rayon/) for more.
```

Comment:
Does this no longer have any effect? (If it still has an effect, we should leave it here, even if `UV_CONCURRENT_INSTALLS` is recommended, since this is intended to document env variables that could affect behavior.)

Reply:
It shouldn't have any effect because we're initialize the global thread pool manually.